optimade/models/jsonapi.py

```diff
@@ -252,7 +255,7 @@ class Resource(BaseResource):
     )
     relationships: Optional[Relationships] = Field(
         None,
-        description="a relationships object describing relationships between the resource and other JSON:API resources.",
+        description="a relationships object describing relationships between the resource and other JSON API resources.",
```

Comment:
Just to make it consistent
```suggestion
        description="A relationships object describing relationships between the resource and other JSON API resources.",
```